src/Views/Gestures.vala

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright (c) 2011-2020 elementary, Inc. (https://elementary.io)
+ *               2020 José Expósito <jose.exposito89@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program; if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ */
+
+public class MouseTouchpad.GesturesView : Granite.SimpleSettingsPage {
+    private Gtk.Switch multitasking_switch;
+    private Gtk.ComboBoxText multitasking_combobox;
+
+    private Gtk.Switch workspaces_switch;
+    private Gtk.ComboBoxText workspaces_combobox;
+
+    private SettingLabel maximize_label;
+    private Gtk.Switch maximize_switch;
+    private Gtk.ComboBoxText maximize_combobox;
+
+    private SettingLabel tile_label;
+    private Gtk.Switch tile_switch;
+    private Gtk.ComboBoxText tile_combobox;
+
+    private GLib.Settings glib_settings = new GLib.Settings ("io.elementary.desktop.wm.gestures");
```

Comment:
We don't usually instantiate objects during variable declaration like this, but also you should check that this settings path exists before you try to instantiate it or you'll get a segfault

Reply:
What should be the fallback in case the path doesn't exists? Disabled UI like what it displayed when Touchégg is not installed? Or a "Not available" splash?
A not available splash could make sense for those packaging Pantheon in other distros...

Reply:
I guess my first instinct would be to just not show the item in the sidebar, but I'd be interested in feedback from @cassidyjames and I guess @decathorpe since he'd be experiencing this in Fedora

Reply:
We usually just don't show the UI when it doesn't exist because that should never happen for a user, or if it does, it's not their fault that their OS/packaging/whatever doesn't support the festure.

Reply:
I'm fine with either option. Not showing the UI if the requirements are missing is probably the "cleanest" solution, but might also confuse users who expect something to be there, and a simple "Disabled because touchegg is not installed" screen is actually actionable.

Reply:
Ok, so I updated the code to reflect your suggestions. This is how it works:

 - If Touchégg is not installed (config file not found), or Gala does not support gestures (GSettings schema not found) the UI is not displayed.
 - If there is a problem parsing the user config file, because they have been messing around with their preferences or whatever, the maximize and tile comboboxes are disabled.